distribution/lib/Standard/Base/0.0.0-dev/package.yaml

```diff
@@ -8,3 +8,8 @@ authors:
 maintainers:
   - name: Enso Team
     email: contact@enso.org
+component-groups:
+  new:
+    - Testme 1:
+      exports:
+        - Main.Nothing
```

Comment:
Change unrelated to the PR title and description

Reply:
Ouch; thanks, and sorry.